src/App.tsx

```diff
@@ -1,24 +1,159 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { ErrorNotification } from './components/ErrorNotification';
+import { ErrorType } from './types/ErrorType';
+import { FilterBy } from './types/FilterBy';
 
-const USER_ID = 0;
+const USER_ID = 6481;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState(ErrorType.None);
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+  const [filterBy, setFilterBy] = useState(FilterBy.All);
+
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isInputActive, setIsInputActive] = useState(true);
+
+  const [updatingId, setUpdatingId] = useState([0]);
```

Comment:
It should be number according naming. Change naming or variable type
```suggestion
  const [updatingId, setUpdatingId] = useState([0]);
```